--- git status ---
On branch dev/humanoid_run
Your branch is up to date with 'origin/dev/humanoid_run'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-311.pyc
	modified:   source/robot_lab/robot_lab/assets/urdf0924.py
	modified:   source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc
	modified:   source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/flat_env_cfg.cpython-311.pyc
	modified:   source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/rough_env_cfg.cpython-311.pyc

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/urdf_0924_flat/2025-11-24_13-47-45/
	outputs/2025-11-24/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-311.pyc b/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-311.pyc
index 9f6784f..92a76b5 100644
Binary files a/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-311.pyc and b/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-311.pyc differ
diff --git a/source/robot_lab/robot_lab/assets/urdf0924.py b/source/robot_lab/robot_lab/assets/urdf0924.py
index a2b3f7c..f6d0138 100644
--- a/source/robot_lab/robot_lab/assets/urdf0924.py
+++ b/source/robot_lab/robot_lab/assets/urdf0924.py
@@ -39,7 +39,7 @@ URDF_0924_CFG = ArticulationCfg(
             max_depenetration_velocity=10.0,
         ),
         articulation_props=sim_utils.ArticulationRootPropertiesCfg(
-            enabled_self_collisions=False, solver_position_iteration_count=4, solver_velocity_iteration_count=0
+            enabled_self_collisions=True, solver_position_iteration_count=4, solver_velocity_iteration_count=0
         ),
         # # for xacro
         # func=spawn_from_lazy_usd,
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc
index 84ff059..224df23 100644
Binary files a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc and b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_cfg.cpython-311.pyc differ
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/flat_env_cfg.cpython-311.pyc b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/flat_env_cfg.cpython-311.pyc
index 438bfa9..137a41d 100644
Binary files a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/flat_env_cfg.cpython-311.pyc and b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/flat_env_cfg.cpython-311.pyc differ
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/rough_env_cfg.cpython-311.pyc b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/rough_env_cfg.cpython-311.pyc
index 3ebabfc..93d9d7f 100644
Binary files a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/rough_env_cfg.cpython-311.pyc and b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/rough_env_cfg.cpython-311.pyc differ