--- git status ---
On branch dev/SEAG_2_URDF
Your branch is up to date with 'origin/dev/SEAG_2_URDF'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/robot_lab/robot_lab/assets/SEAG_2_URDF.py
	modified:   source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/flat_env_cfg.py
	modified:   source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/rough_env_cfg.py
	modified:   source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agents/cusrl_ppo_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/SEAG_2_URDF_rough/
	outputs/2025-12-09/22-02-54/
	outputs/2025-12-09/22-20-12/
	outputs/2025-12-09/22-26-16/
	outputs/2025-12-09/22-34-19/
	outputs/2025-12-09/22-36-21/
	source/robot_lab/robot_lab/assets/__pycache__/SEAG_2_URDF.cpython-311.pyc
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/__pycache__/rough_env_cfg.cpython-311.pyc
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/agents/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/robot_lab/robot_lab/assets/SEAG_2_URDF.py b/source/robot_lab/robot_lab/assets/SEAG_2_URDF.py
index a43ed57..9910b7e 100644
--- a/source/robot_lab/robot_lab/assets/SEAG_2_URDF.py
+++ b/source/robot_lab/robot_lab/assets/SEAG_2_URDF.py
@@ -60,24 +60,14 @@ SEAG_2_URDF_CFG = ArticulationCfg(
     # ... other configuration parameters ...
     init_state=ArticulationCfg.InitialStateCfg(
         pos=(0.0, 0.0, 0.88),
-        joint_pos={ # 27 DOF
-            ".*_hip_pitch_joint": -0.1,
-            ".*_hip_roll_joint": 0.0,
-            ".*_hip_yaw_joint": 0.0,
-            ".*_knee_joint": 0.3,
-            ".*_ankle_pitch_joint": -0.2,
-            ".*_ankle_roll_joint": 0.0,
+        joint_pos={ # 12 DOF
+            ".*_hip_pitch_Joint": -0.1,
+            ".*_hip_roll_Joint": 0.0,
+            ".*_hip_yaw_Joint": 0.0,
+            ".*_knee_pitch_Joint": 0.3,
+            ".*_ankle_pitch_Joint": -0.2,
+            ".*_ankle_roll_Joint": 0.0,
             
-            "torso_joint": 0.0,  # torso joints
-            ".*_shoulder_pitch_joint": -0.0,
-            "right_shoulder_roll_joint": -0.2, # (right -0.2, left 0.2)
-            "left_shoulder_roll_joint": 0.2,
-            ".*_shoulder_yaw_joint": 0.0,
-            ".*_elbow_joint": 0.0,
-            
-            ".*_wrist_roll_joint": 0.0,
-            ".*_wrist_pitch_joint": 0.0,
-            ".*_wrist_yaw_joint": 0.0,
         },
         joint_vel={".*": 0.0},
     ),
@@ -85,69 +75,37 @@ SEAG_2_URDF_CFG = ArticulationCfg(
     actuators={
         "legs": ImplicitActuatorCfg(
             joint_names_expr=[
-                ".*_hip_pitch_joint",
-                ".*_hip_roll_joint",
-                ".*_hip_yaw_joint",
-                ".*_knee_joint",
+                ".*_hip_pitch_Joint",
+                ".*_hip_roll_Joint",
+                ".*_hip_yaw_Joint",
+                ".*_knee_pitch_Joint",
             ],
             effort_limit_sim=300,
             velocity_limit_sim=100.0,
             stiffness={
-                ".*_hip_pitch_joint": 200.0,
-                ".*_hip_roll_joint": 150.0,
-                ".*_hip_yaw_joint": 150.0,
-                ".*_knee_joint": 200.0,
+                ".*_hip_pitch_Joint": 200.0,
+                ".*_hip_roll_Joint": 150.0,
+                ".*_hip_yaw_Joint": 150.0,
+                ".*_knee_pitch_Joint": 200.0,
             },
             damping={
-                ".*_hip_pitch_joint": 5.0,
-                ".*_hip_roll_joint": 5.0,
-                ".*_hip_yaw_joint": 5.0,
-                ".*_knee_joint": 5.0,
+                ".*_hip_pitch_Joint": 5.0,
+                ".*_hip_roll_Joint": 5.0,
+                ".*_hip_yaw_Joint": 5.0,
+                ".*_knee_pitch_Joint": 5.0,
             },
             armature={
                 ".*_hip_.*": 0.01,
-                ".*_knee_joint": 0.01,
-            },
-        ),
-        "waist": ImplicitActuatorCfg(
-            joint_names_expr=["torso_joint"],
-            effort_limit_sim=300,
-            velocity_limit_sim=100.0,
-            stiffness={
-                "torso_joint": 200.0,
-            },
-            damping={
-                "torso_joint": 5.0,
-            },
-            armature={
-                "torso_joint": 0.01,
+                ".*_knee_pitch_Joint": 0.01,
             },
         ),
         "feet": ImplicitActuatorCfg(
             effort_limit_sim=20,
-            joint_names_expr=[".*_ankle_pitch_joint", ".*_ankle_roll_joint"],
+            joint_names_expr=[".*_ankle_pitch_Joint", ".*_ankle_roll_Joint"],
             stiffness=20.0,
             damping=2.0,
             armature=0.01,
         ),
-        "arms": ImplicitActuatorCfg(
-            joint_names_expr=[
-                ".*_shoulder_pitch_joint",
-                ".*_shoulder_roll_joint",
-                ".*_shoulder_yaw_joint",
-                ".*_elbow_joint",
-                ".*_wrist_.*",
-            ],
-            effort_limit_sim=300,
-            velocity_limit_sim=100.0,
-            stiffness=40.0,
-            damping=10.0,
-            armature={
-                ".*_shoulder_.*": 0.01,
-                ".*_elbow_.*": 0.01,
-                ".*_wrist_.*": 0.01,
-            },
-        ),
     },
 )
 
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/flat_env_cfg.py b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/flat_env_cfg.py
index eccc2e4..9a54f76 100644
--- a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/flat_env_cfg.py
+++ b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/flat_env_cfg.py
@@ -32,7 +32,7 @@ class SEAG_2_URDFFlatEnvCfg(SEAG_2_URDFRoughEnvCfg):
         self.rewards.feet_air_time.weight = 0.75
         self.rewards.feet_air_time.params["threshold"] = 1.5
         self.rewards.joint_torques_l2.weight = -2.0e-6
-        self.rewards.joint_torques_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_joint"]
+        self.rewards.joint_torques_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_Joint"]
 
         # If the weight of rewards is 0, set rewards to None
         if self.__class__.__name__ == "SEAG_2_URDFFlatEnvCfg":
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/rough_env_cfg.py b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/rough_env_cfg.py
index 01102ff..ca4d298 100644
--- a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/rough_env_cfg.py
+++ b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/SEAG_2_URDF/rough_env_cfg.py
@@ -13,41 +13,22 @@ from robot_lab.tasks.manager_based.locomotion.velocity.velocity_env_cfg import L
 from robot_lab.assets.SEAG_2_URDF import SEAG_2_URDF_CFG  # isort: skip
 @configclass
 class SEAG_2_URDFRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
-    base_link_name = "pelvis"
-    foot_link_name = ".*_ankle_roll_link"
+    base_link_name = "base_link"
+    foot_link_name = ".*_ankle_roll_Link"
     # fmt: off
     joint_names = [
-        "left_hip_pitch_joint",          # 0  L_LEG_HIP_PITCH
-        "left_hip_roll_joint",           # 1  L_LEG_HIP_ROLL
-        "left_hip_yaw_joint",            # 2  L_LEG_HIP_YAW
-        "left_knee_joint",               # 3  L_LEG_KNEE
-        "left_ankle_pitch_joint",        # 4  L_LEG_ANKLE_B
-        "left_ankle_roll_joint",         # 5  L_LEG_ANKLE_A
-        "right_hip_pitch_joint",         # 6  R_LEG_HIP_PITCH
-        "right_hip_roll_joint",          # 7  R_LEG_HIP_ROLL
-        "right_hip_yaw_joint",           # 8  R_LEG_HIP_YAW
-        "right_knee_joint",              # 9  R_LEG_KNEE
-        "right_ankle_pitch_joint",       # 10 R_LEG_ANKLE_B
-        "right_ankle_roll_joint",        # 11 R_LEG_ANKLE_A
-        
-        "torso_joint",                   # 12 WAIST_YAW
-        
-        "left_shoulder_pitch_joint",     # 13 L_SHOULDER_PITCH
-        "left_shoulder_roll_joint",      # 14 L_SHOULDER_ROLL
-        "left_shoulder_yaw_joint",       # 15 L_SHOULDER_YAW
-        "left_elbow_joint",              # 16 L_ELBOW
-        
-        "right_shoulder_pitch_joint",    # 17 R_SHOULDER_PITCH
-        "right_shoulder_roll_joint",     # 18 R_SHOULDER_ROLL
-        "right_shoulder_yaw_joint",      # 19 R_SHOULDER_YAW
-        "right_elbow_joint",             # 20 R_ELBOW
-        
-        "left_wrist_yaw_joint",         # 21 L_WRIST_YAW           
-        "left_wrist_pitch_joint",       # 22 L_WRIST_PITCH
-        "left_wrist_roll_joint",        # 23 L_WRIST_ROLL
-        "right_wrist_yaw_joint",        # 24 R_WRIST_YAW
-        "right_wrist_pitch_joint",      # 25 R_WRIST_PITCH
-        "right_wrist_roll_joint",       # 26 R_WRIST_ROLL
+        "L_hip_pitch_Joint",          # 0  L_LEG_HIP_PITCH
+        "L_hip_roll_Joint",           # 1  L_LEG_HIP_ROLL
+        "L_hip_yaw_Joint",            # 2  L_LEG_HIP_YAW
+        "L_knee_pitch_Joint",               # 3  L_LEG_KNEE
+        "L_ankle_pitch_Joint",        # 4  L_LEG_ANKLE_B
+        "L_ankle_roll_Joint",         # 5  L_LEG_ANKLE_A
+        "R_hip_pitch_Joint",         # 6  R_LEG_HIP_PITCH
+        "R_hip_roll_Joint",          # 7  R_LEG_HIP_ROLL
+        "R_hip_yaw_Joint",           # 8  R_LEG_HIP_YAW
+        "R_knee_pitch_Joint",              # 9  R_LEG_KNEE
+        "R_ankle_pitch_Joint",       # 10 R_LEG_ANKLE_B
+        "R_ankle_roll_Joint",        # 11 R_LEG_ANKLE_A
     ]
     # fmt: on
 
@@ -97,25 +78,25 @@ class SEAG_2_URDFRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
 
         # Joint penalties
         self.rewards.joint_torques_l2.weight = -1.5e-7
-        self.rewards.joint_torques_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_joint", ".*_ankle_.*"]
+        self.rewards.joint_torques_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_pitch_Joint", ".*_ankle_.*"]
         self.rewards.joint_vel_l2.weight = 0.0
         self.rewards.joint_acc_l2.weight = -1.25e-7
-        self.rewards.joint_acc_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_joint", ".*_ankle_.*"]
+        self.rewards.joint_acc_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_pitch_Joint", ".*_ankle_.*"]
         self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_hip_l1", -0.15, [".*hip_yaw.*", ".*hip_roll.*"])
-        self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_arms_l1", -0.1, [".*shoulder.*", ".*elbow.*"])
-        self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_torso_l1", -0.1, ["torso_joint"])
+        # self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_arms_l1", -0.1, [".*shoulder.*", ".*elbow.*"])
+        # self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_torso_l1", -0.1, ["torso_joint"])
         self.rewards.joint_pos_limits.weight = -0.5
         self.rewards.joint_vel_limits.weight = 0
         self.rewards.joint_power.weight = 0
         self.rewards.stand_still_without_cmd.weight = 0.3
         self.rewards.joint_pos_penalty.weight = -1.0
         self.rewards.joint_mirror.weight = 0.0
-        self.rewards.joint_mirror.params["mirror_joints"] = [["left_(hip|knee|ankle).*", "right_(hip|knee|ankle).*"]]
+        self.rewards.joint_mirror.params["mirror_joints"] = [["L_(hip|knee|ankle).*", "R_(hip|knee|ankle).*"]]
 
         # Action penalties
         self.rewards.action_rate_l2.weight = -0.005
         self.rewards.action_mirror.weight = 0
-        self.rewards.action_mirror.params["mirror_joints"] = [["left_(hip|knee|ankle).*", "right_(hip|knee|ankle).*"]]
+        self.rewards.action_mirror.params["mirror_joints"] = [["L_(hip|knee|ankle).*", "R_(hip|knee|ankle).*"]]
 
         # Contact sensor
         self.rewards.undesired_contacts.weight = 0
@@ -154,7 +135,7 @@ class SEAG_2_URDFRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
         self.rewards.feet_horizontal_simple.params["asset_cfg"].body_names = [self.foot_link_name]
 
         # If the weight of rewards is 0, set rewards to None
-        if self.__class__.__name__ == "URDF_0924RoughEnvCfg":
+        if self.__class__.__name__ == "SEAG_2_URDFRoughEnvCfg":
             self.disable_zero_weight_rewards()
 
         # ------------------------------Terminations------------------------------
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agents/cusrl_ppo_cfg.py b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agents/cusrl_ppo_cfg.py
index d702d38..0bbe900 100644
--- a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agents/cusrl_ppo_cfg.py
+++ b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agents/cusrl_ppo_cfg.py
@@ -7,10 +7,10 @@ from isaaclab.utils import configclass
 
 
 @configclass
-class URDF0924RoughTrainerCfg:
+class SEAG_2_URDFRoughTrainerCfg:
     max_iterations = 3000
     save_interval = 100
-    experiment_name = "urdf_0924_rough"
+    experiment_name = "SEAG_2_URDF_rough"
     agent_factory = cusrl.ActorCritic.Factory(
         num_steps_per_update=24,
         actor_factory=cusrl.Actor.Factory(
@@ -42,8 +42,8 @@ class URDF0924RoughTrainerCfg:
 
 
 @configclass
-class URDF0924FlatTrainerCfg(URDF0924RoughTrainerCfg):
+class SEAG_2_URDFFlatTrainerCfg(SEAG_2_URDFRoughTrainerCfg):
     def __post_init__(self):
         super().__post_init__()
         self.max_iterations = 1500
-        self.experiment_name = "urdf_0924_rough"
+        self.experiment_name = "SEAG_2_URDF_rough"