--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/robot_lab/robot_lab/assets/urdf0924.py
	deleted:    source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/__init__.py
	deleted:    source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/flat_env_cfg.py
	deleted:    source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/rough_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/urdf_0924_flat/
	outputs/2025-09-25/11-27-25/
	outputs/2025-09-25/11-29-58/
	outputs/2025-09-25/11-32-01/
	outputs/2025-09-25/11-33-28/
	source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-310.pyc
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__init__.py
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/__pycache__/
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agents/
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/flat_env_cfg.py
	source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/rough_env_cfg.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/robot_lab/robot_lab/assets/urdf0924.py b/source/robot_lab/robot_lab/assets/urdf0924.py
index b0b2e28..01bfe26 100644
--- a/source/robot_lab/robot_lab/assets/urdf0924.py
+++ b/source/robot_lab/robot_lab/assets/urdf0924.py
@@ -68,7 +68,7 @@ URDF_0924_CFG = ArticulationCfg(
             ".*_ankle_pitch_joint": -0.2,
             ".*_ankle_roll_joint": 0.0,
             
-            ".*_joint": 0.0,  # torso joints
+            "torso_joint": 0.0,  # torso joints
             ".*_shoulder_pitch_joint": 0.0,
             ".*_shoulder_roll_joint": 0.0,
             ".*_shoulder_yaw_joint": 0.0,
@@ -109,7 +109,7 @@ URDF_0924_CFG = ArticulationCfg(
             },
         ),
         "waist": ImplicitActuatorCfg(
-            joint_names_expr=[".*_joint"],
+            joint_names_expr=["torso_joint"],
             effort_limit_sim=300,
             velocity_limit_sim=100.0,
             stiffness={
@@ -124,7 +124,7 @@ URDF_0924_CFG = ArticulationCfg(
         ),
         "feet": ImplicitActuatorCfg(
             effort_limit_sim=20,
-            joint_names_expr=["*_knee_joint",".*_ankle_pitch_joint", ".*_ankle_roll_joint"],
+            joint_names_expr=[".*_knee_joint",".*_ankle_pitch_joint", ".*_ankle_roll_joint"],
             stiffness=20.0,
             damping=2.0,
             armature=0.01,
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/__init__.py b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/__init__.py
deleted file mode 100644
index 7e10e3c..0000000
--- a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/__init__.py
+++ /dev/null
@@ -1,33 +0,0 @@
-﻿# Copyright (c) 2024-2025 Ziqi Fan
-# SPDX-License-Identifier: Apache-2.0
-
-import gymnasium as gym
-
-from . import agents
-
-##
-# Register Gym environments.
-##
-
-gym.register(
-    id="RobotLab-Isaac-Velocity-Rough-URDF0924-v0",
-    entry_point="isaaclab.envs:ManagerBasedRLEnv",
-    disable_env_checker=True,
-    kwargs={
-        "env_cfg_entry_point": f"{__name__}.rough_env_cfg:URDF_0924RoughEnvCfg",
-        "rsl_rl_cfg_entry_point": f"{agents.__name__}.rsl_rl_ppo_cfg:UnitreeG1RoughPPORunnerCfg", #
-        "cusrl_cfg_entry_point": f"{agents.__name__}.cusrl_ppo_cfg:UnitreeG1RoughTrainerCfg",#
-    },
-)
-
-
-gym.register(
-    id="RobotLab-Isaac-Velocity-Flat-URDF0924-v0",
-    entry_point="isaaclab.envs:ManagerBasedRLEnv",
-    disable_env_checker=True,
-    kwargs={
-        "env_cfg_entry_point": f"{__name__}.flat_env_cfg:URDF_0924FlatEnvCfg",
-        "rsl_rl_cfg_entry_point": f"{agents.__name__}.rsl_rl_ppo_cfg:UnitreeG1FlatPPORunnerCfg",
-        "cusrl_cfg_entry_point": f"{agents.__name__}.cusrl_ppo_cfg:UnitreeG1FlatTrainerCfg",
-    },
-)
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/flat_env_cfg.py b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/flat_env_cfg.py
deleted file mode 100644
index c9566db..0000000
--- a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/flat_env_cfg.py
+++ /dev/null
@@ -1,41 +0,0 @@
-﻿# Copyright (c) 2024-2025 Ziqi Fan
-# SPDX-License-Identifier: Apache-2.0
-
-from isaaclab.utils import configclass
-
-from .rough_env_cfg import URDF_0924RoughEnvCfg
-
-
-@configclass
-class URDF_0924FlatEnvCfg(URDF_0924RoughEnvCfg):
-    def __post_init__(self):
-        # post init of parent
-        super().__post_init__()
-
-        # override rewards
-        self.rewards.base_height_l2.params["sensor_cfg"] = None
-        # change terrain to flat
-        self.scene.terrain.terrain_type = "plane"
-        self.scene.terrain.terrain_generator = None
-        # no height scan
-        self.scene.height_scanner = None
-        self.observations.policy.height_scan = None
-        self.observations.critic.height_scan = None
-        # no terrain curriculum
-        self.curriculum.terrain_levels = None
-
-        # Rewards
-        self.rewards.track_ang_vel_z_exp.weight = 1.0
-        self.rewards.lin_vel_z_l2.weight = -0.2
-        self.rewards.action_rate_l2.weight = -0.005
-        self.rewards.joint_acc_l2.weight = -1.0e-7
-        self.rewards.feet_air_time.weight = 0.75
-        self.rewards.feet_air_time.params["threshold"] = 0.4
-        self.rewards.joint_torques_l2.weight = -2.0e-6
-        self.rewards.joint_torques_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_joint"]
-
-        # If the weight of rewards is 0, set rewards to None
-        if self.__class__.__name__ == "URDF_0924FlatEnvCfg":
-            self.disable_zero_weight_rewards()
-
-
diff --git a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/rough_env_cfg.py b/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/rough_env_cfg.py
deleted file mode 100644
index 6aa9d41..0000000
--- a/source/robot_lab/robot_lab/tasks/manager_based/locomotion/velocity/config/humanoid/urdf0924/agent/rough_env_cfg.py
+++ /dev/null
@@ -1,169 +0,0 @@
-﻿# Copyright (c) 2024-2025 Ziqi Fan
-# SPDX-License-Identifier: Apache-2.0
-
-from isaaclab.utils import configclass
-
-import robot_lab.tasks.manager_based.locomotion.velocity.mdp as mdp
-from robot_lab.tasks.manager_based.locomotion.velocity.velocity_env_cfg import LocomotionVelocityRoughEnvCfg
-
-##
-# Pre-defined configs
-##
-# from isaaclab_assets.robots.unitree import G1_MINIMAL_CFG  # isort: skip
-from robot_lab.assets.urdf0924 import URDF_0924_CFG  # isort: skip
-
-@configclass
-class URDF_0924RoughEnvCfg(LocomotionVelocityRoughEnvCfg):
-    base_link_name = "pelvis"
-    foot_link_name = ".*_ankle_roll_link"
-    # fmt: off
-    joint_names = [
-        "left_hip_pitch_joint",          # 0  L_LEG_HIP_PITCH
-        "left_hip_roll_joint",           # 1  L_LEG_HIP_ROLL
-        "left_hip_yaw_joint",            # 2  L_LEG_HIP_YAW
-        "left_knee_joint",               # 3  L_LEG_KNEE
-        "left_ankle_pitch_joint",        # 4  L_LEG_ANKLE_B
-        "left_ankle_roll_joint",         # 5  L_LEG_ANKLE_A
-        "right_hip_pitch_joint",         # 6  R_LEG_HIP_PITCH
-        "right_hip_roll_joint",          # 7  R_LEG_HIP_ROLL
-        "right_hip_yaw_joint",           # 8  R_LEG_HIP_YAW
-        "right_knee_joint",              # 9  R_LEG_KNEE
-        "right_ankle_pitch_joint",       # 10 R_LEG_ANKLE_B
-        "right_ankle_roll_joint",        # 11 R_LEG_ANKLE_A
-        
-        "torso_joint",                   # 12 WAIST_YAW
-        
-        "left_shoulder_pitch_joint",     # 13 L_SHOULDER_PITCH
-        "left_shoulder_roll_joint",      # 14 L_SHOULDER_ROLL
-        "left_shoulder_yaw_joint",       # 15 L_SHOULDER_YAW
-        "left_elbow_joint",              # 16 L_ELBOW
-        
-        "right_shoulder_pitch_joint",    # 17 R_SHOULDER_PITCH
-        "right_shoulder_roll_joint",     # 18 R_SHOULDER_ROLL
-        "right_shoulder_yaw_joint",      # 19 R_SHOULDER_YAW
-        "right_elbow_joint",             # 20 R_ELBOW
-        
-        "left_wrist_yaw_joint",         # 21 L_WRIST_YAW           
-        "left_wrist_pitch_joint",       # 22 L_WRIST_PITCH
-        "left_wrist_roll_joint",        # 23 L_WRIST_ROLL
-        "right_wrist_yaw_joint",        # 24 R_WRIST_YAW
-        "right_wrist_pitch_joint",      # 25 R_WRIST_PITCH
-        "right_wrist_roll_joint",       # 26 R_WRIST_ROLL
-    ]
-    # fmt: on
-
-    def __post_init__(self):
-        # post init of parent
-        super().__post_init__()
-
-        # ------------------------------Sence------------------------------
-        self.scene.robot = URDF_0924_CFG.replace(prim_path="{ENV_REGEX_NS}/Robot")
-        self.scene.height_scanner.prim_path = "{ENV_REGEX_NS}/Robot/" + self.base_link_name
-        self.scene.height_scanner_base.prim_path = "{ENV_REGEX_NS}/Robot/" + self.base_link_name
-
-        # ------------------------------Observations------------------------------
-        self.observations.policy.base_lin_vel.scale = 2.0
-        self.observations.policy.base_ang_vel.scale = 0.25
-        self.observations.policy.joint_pos.scale = 1.0
-        self.observations.policy.joint_vel.scale = 0.05
-        self.observations.policy.base_lin_vel = None
-        self.observations.policy.height_scan = None
-        self.observations.policy.joint_pos.params["asset_cfg"].joint_names = self.joint_names
-        self.observations.policy.joint_vel.params["asset_cfg"].joint_names = self.joint_names
-
-        # ------------------------------Actions------------------------------
-        # reduce action scale
-        self.actions.joint_pos.scale = 0.25
-        self.actions.joint_pos.clip = {".*": (-100.0, 100.0)}
-        self.actions.joint_pos.joint_names = self.joint_names
-
-        # ------------------------------Events------------------------------
-        self.events.randomize_rigid_body_mass.params["asset_cfg"].body_names = [self.base_link_name]
-        self.events.randomize_com_positions.params["asset_cfg"].body_names = [self.base_link_name]
-        self.events.randomize_apply_external_force_torque.params["asset_cfg"].body_names = [self.base_link_name]
-
-        # ------------------------------Rewards------------------------------
-        # General
-        self.rewards.is_terminated.weight = -200.0
-
-        # Root penalties
-        self.rewards.lin_vel_z_l2.weight = 0
-        self.rewards.ang_vel_xy_l2.weight = -0.1
-        self.rewards.flat_orientation_l2.weight = -0.2
-        self.rewards.base_height_l2.weight = 0
-        self.rewards.base_height_l2.params["target_height"] = 0
-        self.rewards.base_height_l2.params["asset_cfg"].body_names = [self.base_link_name]
-        self.rewards.body_lin_acc_l2.weight = 0
-        self.rewards.body_lin_acc_l2.params["asset_cfg"].body_names = [self.base_link_name]
-
-        # Joint penalties
-        self.rewards.joint_torques_l2.weight = -1.5e-7
-        self.rewards.joint_torques_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_joint", ".*_ankle_.*"]
-        self.rewards.joint_vel_l2.weight = 0
-        self.rewards.joint_acc_l2.weight = -1.25e-7
-        self.rewards.joint_acc_l2.params["asset_cfg"].joint_names = [".*_hip_.*", ".*_knee_joint"]
-        self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_hip_l1", -0.1, [".*hip_yaw.*", ".*hip_roll.*"])
-        self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_arms_l1", -0.1, [".*shoulder.*", ".*elbow.*"])
-        self.rewards.create_joint_deviation_l1_rewterm("joint_deviation_torso_l1", -0.1, ["torso_joint"])
-        self.rewards.joint_pos_limits.weight = -0.5
-        self.rewards.joint_vel_limits.weight = 0
-        self.rewards.joint_power.weight = 0
-        self.rewards.stand_still_without_cmd.weight = 0
-        self.rewards.joint_pos_penalty.weight = -1.0
-        self.rewards.joint_mirror.weight = 0
-        self.rewards.joint_mirror.params["mirror_joints"] = [["left_(hip|knee|ankle).*", "right_(hip|knee|ankle).*"]]
-
-        # Action penalties
-        self.rewards.action_rate_l2.weight = -0.005
-        self.rewards.action_mirror.weight = 0
-        self.rewards.action_mirror.params["mirror_joints"] = [["left_(hip|knee|ankle).*", "right_(hip|knee|ankle).*"]]
-
-        # Contact sensor
-        self.rewards.undesired_contacts.weight = 0
-        self.rewards.undesired_contacts.params["sensor_cfg"].body_names = [f"^(?!.*{self.foot_link_name}).*"]
-        self.rewards.contact_forces.weight = 0
-        self.rewards.contact_forces.params["sensor_cfg"].body_names = [self.foot_link_name]
-
-        # Velocity-tracking rewards
-        self.rewards.track_lin_vel_xy_exp.weight = 3.0
-        self.rewards.track_lin_vel_xy_exp.func = mdp.track_lin_vel_xy_yaw_frame_exp
-        self.rewards.track_ang_vel_z_exp.weight = 3.0
-        self.rewards.track_ang_vel_z_exp.func = mdp.track_ang_vel_z_world_exp
-
-        # Others
-        self.rewards.feet_air_time.weight = 0.25
-        self.rewards.feet_air_time.func = mdp.feet_air_time_positive_biped
-        self.rewards.feet_air_time.params["threshold"] = 0.4
-        self.rewards.feet_air_time.params["sensor_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_contact.weight = 0
-        self.rewards.feet_contact.params["sensor_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_contact_without_cmd.weight = 0
-        self.rewards.feet_contact_without_cmd.params["sensor_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_stumble.weight = 0
-        self.rewards.feet_stumble.params["sensor_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_slide.weight = -0.2
-        self.rewards.feet_slide.params["sensor_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_slide.params["asset_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_height.weight = 0
-        self.rewards.feet_height.params["target_height"] = 0.05
-        self.rewards.feet_height.params["asset_cfg"].body_names = [self.foot_link_name]
-        self.rewards.feet_height_body.weight = 0
-        self.rewards.feet_height_body.params["target_height"] = -0.2
-        self.rewards.feet_height_body.params["asset_cfg"].body_names = [self.foot_link_name]
-        self.rewards.upward.weight = 1.0
-
-        # If the weight of rewards is 0, set rewards to None
-        if self.__class__.__name__ == "UnitreeG1RoughEnvCfg":
-            self.disable_zero_weight_rewards()
-
-        # ------------------------------Terminations------------------------------
-        self.terminations.illegal_contact.params["sensor_cfg"].body_names = [self.base_link_name]
-
-        # ------------------------------Curriculums------------------------------
-        # self.curriculum.command_levels.params["range_multiplier"] = (0.2, 1.0)
-        self.curriculum.command_levels = None
-
-        # ------------------------------Commands------------------------------
-        self.commands.base_velocity.ranges.lin_vel_x = (-1.0, 1.0)
-        self.commands.base_velocity.ranges.lin_vel_y = (-1.0, 1.0)
-        self.commands.base_velocity.ranges.ang_vel_z = (-1.0, 1.0)