--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   logs/rsl_rl/unitree_h1_flat/2025-09-24_17-44-47/exported/policy.pt
	modified:   source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-310.pyc
	modified:   source/robot_lab/robot_lab/assets/urdf0924.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/urdf_0924_flat/2025-09-25_12-04-04/
	logs/rsl_rl/urdf_0924_flat/2025-09-25_19-44-06/
	outputs/2025-09-25/12-04-04/
	outputs/2025-09-25/18-51-24/
	outputs/2025-09-25/18-53-43/
	outputs/2025-09-25/19-44-06/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/logs/rsl_rl/unitree_h1_flat/2025-09-24_17-44-47/exported/policy.pt b/logs/rsl_rl/unitree_h1_flat/2025-09-24_17-44-47/exported/policy.pt
index 0cdab4b..efd4476 100644
Binary files a/logs/rsl_rl/unitree_h1_flat/2025-09-24_17-44-47/exported/policy.pt and b/logs/rsl_rl/unitree_h1_flat/2025-09-24_17-44-47/exported/policy.pt differ
diff --git a/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-310.pyc b/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-310.pyc
index 6e3c4ce..9e5bbc7 100644
Binary files a/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-310.pyc and b/source/robot_lab/robot_lab/assets/__pycache__/urdf0924.cpython-310.pyc differ
diff --git a/source/robot_lab/robot_lab/assets/urdf0924.py b/source/robot_lab/robot_lab/assets/urdf0924.py
index 01bfe26..5a6719f 100644
--- a/source/robot_lab/robot_lab/assets/urdf0924.py
+++ b/source/robot_lab/robot_lab/assets/urdf0924.py
@@ -59,20 +59,21 @@ URDF_0924_CFG = ArticulationCfg(
     ),
     # ... other configuration parameters ...
     init_state=ArticulationCfg.InitialStateCfg(
-        pos=(0.0, 0.0, 0.8),
+        pos=(0.0, 0.0, 0.83),
         joint_pos={ # 27 DOF
-            ".*_hip_pitch_joint": -0.1,
+            ".*_hip_pitch_joint": -0.4,
             ".*_hip_roll_joint": 0.0,
             ".*_hip_yaw_joint": 0.0,
-            ".*_knee_joint": 0.3,
-            ".*_ankle_pitch_joint": -0.2,
+            ".*_knee_joint": 0.8,
+            ".*_ankle_pitch_joint": -0.4,
             ".*_ankle_roll_joint": 0.0,
             
             "torso_joint": 0.0,  # torso joints
-            ".*_shoulder_pitch_joint": 0.0,
-            ".*_shoulder_roll_joint": 0.0,
+            ".*_shoulder_pitch_joint": -0.3,
+            "right_shoulder_roll_joint": -0.2, #(right -0.2, left 0.2)
+            "left_shoulder_roll_joint": 0.2,
             ".*_shoulder_yaw_joint": 0.0,
-            ".*_elbow_joint": 0.0,
+            ".*_elbow_joint": 0.1,
             
             ".*_wrist_roll_joint": 0.0,
             ".*_wrist_pitch_joint": 0.0,